OracleJava/README.md

```diff
@@ -1,23 +1,30 @@
 Oracle Java on Docker
 =====
-This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux for all the currently available Oracle JDK's and Server JRE 8.
+This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of currently supportes versions 11 and later and for Server JRE 8.
 
 Oracle Java Server JRE provides the features from Oracle Java JDK commonly required for server-side applications (i.e. Running a Java EE application server). For more information about Server JRE, visit the [Understanding the Server JRE blog entry](https://blogs.oracle.com/java-platform-group/understanding-the-server-jre) from the Java Product Management team.
 
 ## Building the Oracle Java base image
-[Download JDK or Server JRE](https://www.oracle.com/in/java/technologies/javase-downloads.html) file as per your requirement from [Oracle Technical Resources](https://www.oracle.com/in/technical-resources/) and place it in the same directory as the Dockerfile.
+Download the linux x-64 compressed archive (tar.gz) [JDK or Server JRE] (https://www.oracle.com/in/java/technologies/javase-downloads.html) for the version you want to create an image of and place it in the same directory as the corresponding Dockerfile.
 
-Navigate to the folder containing the downloads and run:
+e.g. for JDK 14 download jdk-14[X]_linux-x64_bin.tar.gz into OracleJava/14, 
+for ServerJRE 8 download server-jre-8uXXX-linux-x64.tar.gz into OracleJava/8
+
+Navigate to the folder containing the download and run docker build. Tag it with the correct version number.
+
+e.g. For JDK 14 run
```

Comment:
I thought you wanted a generic command, hence my proposal. This looks good. Should we use JDK 15 instead of 14 as that is the version? For OpenJDK we seem to be using JDK 15 as the example.

Reply:
I want my cake and to eat it too! The command should be specific enough that it looks correct. Leaving it with XX without some examples makes it harder to use for a newbie -and only newbies need this spelled out anyway- so I don't want to have the commands with values that the reader has to magically know mean variables.  On the other hand it can't be hard-coded to a single value given that we have several folders.  That is why I went for generic instructions AND specific example.  Since we don't have to update the example on every release its ok to have them be specific.  We could update them to 15 but I wanted to use 14 because 14 get your 14.0.2 but I still tag it as "14".  Since 15 doesn't have more digits in the version now using 15 in the example could lead you to wonder if you should tag the CPUs with the full versions "15.0.1" or leave it as "15".  Either i fine BTW, we will tag the 15.0.1 with "latest", "15", and "15.0.1" but I wanted the example to make it clear that you don't have to mock around with the build.sh text only because you are not using the GA release.